app/helpers/katello/concerns/smart_proxy_helper_extensions.rb

```diff
@@ -30,6 +30,27 @@ def available_percent(percent_string)
         "#{available_percent}%"
       end
 
+      def download_policies
+        policies = [{
+          :name => "On Demand",
+          :label => ::Runcible::Models::YumImporter::DOWNLOAD_ON_DEMAND
+        },
+                    {
```

Comment:
weird, i wonder if rubocop did that via it s auto correction, i certainly didn't